plugins/battery/battery.plugin.zsh

```diff
@@ -72,7 +72,7 @@ elif [[ $(uname) == "Linux"  ]] ; then
 
   function battery_pct() {
     if (( $+commands[acpi] )) ; then
-      echo "$(acpi | cut -f2 -d ',' | tr -cd '[:digit:]')"
+      echo "$(acpi 2&>/dev/null | cut -f2 -d ',' | tr -cd '[:digit:]')"
```

Comment:
The extra ampersand (`&`) isn't needed to redirect stderr and it's not common, though it doesn't hurt. The same applies to the other change.